CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ endif ()
 
 include (cmake/cpu_features.cmake)
 
-set (CMAKE_CXX_STANDARD 17)
+set (CMAKE_CXX_STANDARD 20)
```

Comment:
Can it work under MacOS x86|arm64 as well?

Reply:
Work under MacOS arm64.